front/temporal/usage_queue/activities.ts

```diff
@@ -18,6 +18,7 @@ export async function recordUsageActivity(workspaceId: string) {
   const subscription = await Subscription.findOne({
     where: {
       workspaceId: workspace.id,
+      status: "active",
```

Comment:
We could also do an `Authenticator.internalBuilderForWorkspace(workspace.sId)` to get an auth and fetch the subscription from the auth object. Both work!